changelogs/fragments/6117-remove-stormssh-depend.yml

```
@@ -0,0 +1,2 @@
+trivial:
```

This is **not** a trivial change, but a feature.
```suggestion
minor_changes:
```